src/stdbuf/libstdbuf.rs

```diff
@@ -47,12 +42,12 @@ fn set_buffer(stream: *mut FILE, value: &str) {
 #[no_mangle]
 pub extern fn stdbuf() {
     if let Ok(val) = env::var("_STDBUF_E") {
-        set_buffer(stderr, &val);
+        set_buffer(STDERR_FILENO as *mut FILE, &val);
```

Comment:
I ran it on both OS X and Linux, but neither system was able to show more than:

```
stdbuf: error: Invalid options
Try 'stdbuf --help' for more information.
```

Unfortunately, this is the same result prior to the changes in the commit. It seems `stdbuf` has been broken for some time. Looking back I should've addressed the errors on Linux before tackling the OS X build.

I'll revert the changes in `libstdbuf.rs`, since I've found the flags needed for clang to compile on OS X. This will help address one issue at a time.

Thanks for reviewing the code.